azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/EntityRuntimeInfo.java

```diff
@@ -13,14 +13,14 @@
     String path;
 
     /**
-     * The path of the entity.
+     * @return Gets the path of the entity.
```

Comment:
>Gets [](start = 15, length = 4)

Remove 'Gets'

Reply:
Also every method description should have something before @return. JavaDoc tool generates documentation with method description followed by parameters and return. In this case, method description is empty. Fix it for every method in this class. The description could be as simple as 'Gets the path of the entiy'

---
In reply to: [260569930](https://github.com/Azure/azure-service-bus-java/pull/341#discussion_r260569930) [](ancestors = 260569930)

Reply:
I did this after reading about what others do in these things. Unless we add something more than what we are writing in `@return` or `@param`, its not worth duplicating the comment in description as well as return/param for getters and setters. 
https://stackoverflow.com/questions/1028967/simple-getter-setter-comments